### PR TITLE
Update corrected areas

### PR DESCRIPTION
**Pull request by ershockley**
## What does the code in this PR do / what does it improve?
This adds functionality for a potential time-dependent S1 correction, like we already have for S2s. It assumes a relative correction, similar to that of the extraction efficiency.

### CODE REVIEW

**Comment by skazama** [comment]
@ershockley maybe we also need to add this correction to `EventInfoDouble`? @DonNabla ?

**Comment by l-althueser** [comment]
Good point, we should add the special case to be consistent.

**Comment by l-althueser** [comment]
And add also the missing cs2 without time correction.

File: straxen/plugins/event_processing.py [hunk 1]
```diff
@@ -659,12 +659,21 @@ class CorrectedAreas(strax.Plugin):
         default='cmt://rel_extraction_eff?version=ONLINE&run_id=plugin.run_id',
         help='Relative extraction efficiency for this run (allows for time dependence)')
 
+    # relative light yield
+    # defaults to no correction
+    rel_light_yield = straxen.URLConfig(
+        default='cmt://relative_light_yield?version=ONLINE&run_id=plugin.run_id',
```
Comment (skazama) [inline straxen/plugins/event_processing.py:665]: sorry @l-althueser and @a1exndr, what's the value for the future data? 1?

Comment (l-althueser) [reply straxen/plugins/event_processing.py:665]: Yes: https://github.com/XENONnT/private_nt_aux_files/pull/152/files#diff-c19121527752e93c5048c5104740d1470ffeb8b4152ce77c836ea3a8a4868e3fR354

Comment (l-althueser) [reply straxen/plugins/event_processing.py:665]: image

**Review by skazama — APPROVED** [review]

**Pull request merged**